Quadriga/src/main/resources/locale/messages_en_US.properties

```diff
@@ -142,13 +142,14 @@ account_approval.error=Sorry, the account couldn't be approved. There seems to b
 account_approval.account_roles.required=Please specify at least one role for the user.
 account_approval.action.required=Please select "Approve" or "Deny".
 
-# project blog entry error messages
-blog_title.required=Please enter title of blog entry.
-blog_description.required=Please enter a text for the new blog entry.
-
-# Text Management
-filename.required = Filename is required.
-filecontent.required = File content is required.
-filename.proper = The filename should only contain the following characters: 0-9, a-Z, _, and -.
-reference.required = A reference is required. Please provide the URI of the original copy of the text.
-empty.textfiles = There are no text files in this workspace.
+# project blog entry error messages
+blog_title.required=Please enter title of blog entry.
+blog_description.required=Please enter a text for the new blog entry.
+
+# Text Management
+filename.required = Filename is required.
+filecontent.required = File content is required.
+filename.proper = The filename should only contain the following characters: 0-9, a-Z, _, and -.
+reference.required = A reference is required. Please provide the URI of the original copy of the text.
+empty.textfiles = There are no text files in this workspace.
+accessibility.required = Please choose a accessibility option for this textfile.
```

Comment:
Should be: "Who should be able to see this text? Everybody (select "Public") or just members of your project (select "Private")?